lib/mayaUsd/fileio/translators/translatorMaterial.cpp

```diff
@@ -449,6 +449,12 @@ bool UsdMayaTranslatorMaterial::AssignMaterial(
                         // identifier is the UUID.
                         subsetRoundtrippingInfo[UsdMayaGeomSubsetTokens->MaterialUuidKey]
                             = JsValue(depNodeFn.uuid().asString().asChar());
+                    } else {
+                        TF_RUNTIME_ERROR(
```

Comment:
Would it be necessary to also have an error for the iterator check above? IDK what are the semantics for the find to fail.

Reply:
Yes, I will revisit the error handling today as this seem a bit fragile.